Restaurant

mel182 opened the pull request.

Commit:
IngredientMenuItem in progress

Commit:
Merge branch 'restaurant' of https://github.com/mrjink/molveno into restaurant

Commit:
Merge remote-tracking branch 'origin/restaurant' into restaurant

marlouvdS committed:
Tablet restaurant page

marlouvdS committed:
Tablet restaurant page

Commit:
Merge remote-tracking branch 'origin/restaurant' into restaurant

# Conflicts:
#	src/main/java/com/molvenolakeresort/models/restaurant/stock/Ingredient.java
#	src/main/java/com/molvenolakeresort/models/restaurant/stock/Stock.java
#	src/main/java/com/molvenolakeresort/models/restaurant/stock/SupplyOrderItem.java

Commit:
StockController

marlouvdS committed:
Tablet restaurant page

marlouvdS committed:
Merge branch 'restaurant' of https://github.com/mrjink/molveno into restaurant

Sandeleen committed:
Made changes in restaurant admin dashboard

Sandeleen committed:
Merge remote-tracking branch 'origin/restaurant' into restaurant

Sandeleen committed:
Added Modify Menu Items

JuulCapgemini committed:
Fixed more ids not being Long. Gave entities a name.

JuulCapgemini committed:
Merge remote-tracking branch 'origin/restaurant' into restaurant

Sandeleen committed:
Merge remote-tracking branch 'origin/restaurant' into restaurant

mel182 committed:
Order ingredient page added

mel182 committed:
Merge remote-tracking branch 'origin/restaurant' into restaurant

marlouvdS committed:
Tablet restaurant page

mel182 committed:
add ingredient order to list page added

mel182 committed:
Merge branch 'restaurant' of https://github.com/mrjink/molveno into restaurant

Sandeleen committed:
Added Modify Menu Items

Sandeleen committed:
Merge remote-tracking branch 'origin/restaurant' into restaurant

Sandeleen committed:
Added link to order ingredient

Sandeleen committed:
Added link to order ingredient

mel182 committed:
Restaurant code clean up

erwinjacobs90 reviewed (approved):
Toppie, goed bezig! ^^

mel182 committed:
Stock issue fixed

Sandeleen reviewed (commented):
Goed bezig!

Commit:
- wat dingen hernoemd.

- fixes voor de relaties. Als het goed is worden nu alle juiste tabellen aangemaak en niet nog eens 10 anderen met hele lange namen.. Moet nog wel zorgvuldig getest worden via Postmaster o.i.d.

Commit:
Merge remote-tracking branch 'origin/restaurant' into restaurant

mel182 committed:
Stock DTO added

mel182 committed:
Merge remote-tracking branch 'origin/restaurant' into restaurant

mel182 committed:
Application properties H2 set

Sandeleen reviewed (approved):
Perfect!
